src/Components/LoadingScreen/LoadingScreen.jsx

```diff
@@ -1,7 +1,128 @@
 import React from "react";
+import styled, { keyframes } from "styled-components";
+import { theme, flexColumnCenter, flexCenter } from "../../styles/theme";
+
+// list component에서 3초? 5초?간 무조건 이 화면 보이게 할것!!
+// 동적 라우팅으로 list component로 연결될것이기 때문에 그 화면에서
+// this.props.match같은것에서 이 component로 필요한 정보 props로 보내줄 것임!!
 
 const LoadingScreen = (props) => {
-  return <div>로딩 스크린(이 될 것)</div>;
+  return (
+    <LoadingScreenContainer>
+      <LoadingBackgroundImg src="https://i.ibb.co/6bd6KWY/30.jpg" />
```

Comment:
```suggestion
      <LoadingBackgroundImg src={BG_IMG} />
      
      ...
// 컴포넌트 바깥 하단
const BG_IMG = "https://i.ibb.co/6bd6KWY/30.jpg"
```
직접 값을 써줄 경우 이 주소가 뜻하는 바가 무엇인지 알기 어려울 수 있습니다. 이 경우 별도의 상수 변수를 선언하고 의미를 부여한 이름을 지어주면 더욱 가독성 높은 코드를 작성할 수 있겠네요!

Reply:
수정하겠습니다!!